tests/sentry/tasks/integrations/github/test_patch_parsers.py

```diff
@@ -20,3 +20,254 @@ def test_python_in_class(self):
             "delete",
             "put",
         }
+
+    def test_javascript_simple(self):
+        patch = """@@44,38@@ function hello(argument1, argument2)
+
+@@44,38@@ export default function world({argument}) {
```

Comment:
Thinking about other test cases here...

Async? What about no args? What if `{` is on next line

```suggestion
export default async function world({argument}) {
```

Reply:
the `@@44, 38@@` is how git represents the hunk header with the line number. anything else is code itself

the regex accounts for any words before the word `function`, but i can add more

i can add a case for { on the next line

Reply:
Oops sorry, wasn't meant to be a github suggestion